src/components/Nav/Nav.js

```diff
@@ -1,22 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useLocation, useHistory } from 'react-router-dom';
-import styled, { css } from 'styled-components';
+import styled from 'styled-components';
 
 import SubNav from './SubNav/SubNav';
+import SignOut from './SignOut/SignOut';
+import SignIn from './SignIn/SignIn';
+
 import { DEALS } from './SubNav/navTabList';
+import { getToken } from '../../utilities/token';
 
-const NEED_SUBNAV_PATH = ['deals', 'my'];
+const NEED_SUBNAV_PATH = ['deals', 'mypage'];
 
 function Nav() {
+  const [isValidUser, setIsValidUser] = useState(false);
+
   const location = useLocation();
   const history = useHistory();
 
+  useEffect(() => {
+    const isValidUser = getToken();
+    setIsValidUser(isValidUser);
```

Comment:
따로 변수에 담을 필요 없을 것 같습니다.
```suggestion
    setIsValidUser(getToken());
```